internal/cli/api/api.go

```diff
@@ -253,7 +253,7 @@ func convertAPIToCobraCommand(command api.Command) (*cobra.Command, error) {
 	addVersionFlag(cmd, command, &version)
 
 	if needsFileFlag(command) {
-		cmd.Flags().StringVar(&file, flag.File, "", "path to the file which contains the api request contents")
+		cmd.Flags().StringVar(&file, flag.File, "", "path to the file which contains the api request contents, alternatively you may pass standard input")
```

Comment:
What about this:
> Path to your API request file. Leave empty to use stdin instead - you must provide one or the other, but not both.

The main reason we're changing this sentence is to:
- Make it clear that either file or stdin is required
- Make it clear that you have to provide one or the other but not both

That was the main feedback we got from the docs team.